Cubical/HITs/SetCoequalizer/Base.agda

```diff
@@ -0,0 +1,23 @@
+{-
+
+This file contains:
+
+- Definition of set coequalizers as performed in https://1lab.dev/Data.Set.Coequaliser.html
+
+-}
+{-# OPTIONS --safe #-}
+module Cubical.HITs.SetCoequalizer.Base where
+
+open import Cubical.Core.Primitives
+
+private
+  variable
+    ℓ ℓ' ℓ'' : Level
+    A : Type ℓ
+    B : Type ℓ'
+
+-- Set coequalizers as a higher inductive type
+data SetCoequalizer {A : Type ℓ} {B : Type ℓ'} (f g : A → B) : Type (ℓ-max ℓ ℓ') where
+  inc    : B → SetCoequalizer f g
+  glue   : (a : A) → inc (f a) ≡ inc (g a)
```

Comment:
I don't know if I like calling this "glue"... There's already so much "glue" in the library. Any suggestions for a better name?

Reply:
'eq' because it (co-)equalizes stuff?

Reply:
Yeah, or `coeq`?

Reply:
Yes. ```coeq``` is the best so far.

Reply:
renamed to coeq